### DISCUSSION TITLE
Paged build failed

### DISCUSSION THREAD

**Discussion by shusen-zhang-ai** [community/community#110935] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I tried to use Github Page to launch a html generated from Simulink. But I got a very ambiguous error message from GitHub Pages. How can I debug with this message. How can I learn what went wrong with Github Pages. The html file already worked with local http server.

[
[temp_statemachine.zip](https://github.com/community/community/files/14487593/temp_statemachine.zip)
](url)

Your site is having problems building: Page build failed. For more information, see https://docs.github.com/enterprise/3.6/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#troubleshooting-build-errors.

**Comment by SebastienXia** [comment]
Hi, @ShusenZhangApp

You have to wait a few minutes after creating the Github page because the Github page takes some time to assimilate your HTML code in the browser.

**Comment by SebastienXia** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> I tried to use Github Page to launch a html generated from Simulink. But I got a very ambiguous error message from GitHub Pages. How can I debug with this message. How can I learn what went wrong with Github Pages. The html file already worked with local http server.
> 
> [ [temp_statemachine.zip](https://github.com/community/community/files/14487593/temp_statemachine.zip) ](url)
> 
> Your site is having problems building: Page build failed. For more information, see https://docs.github.com/enterprise/3.6/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#troubleshooting-build-errors.

Besides, when I try to open your html file, on my computer it shows me this:
![ddd](https://github.com/community/community/assets/105004912/4e28ae9e-7d8b-492f-8c4e-77b37ababc0c)

**Reply by shusen-zhang-ai** [reply]
That html error can be resolved by following the instructions on Windows or Linux. No MATLAB is required. I also tried to use python3 to launch a local http server on 8000 in terminal. It is working. I will try to test the flow on a public domain. It might be due to the custom domain I am in. Thank you for advising!
